assets/base.css

```diff
@@ -3148,6 +3148,7 @@ details-disclosure > details {
 
   .scroll-trigger--active.animate--slide-in {
     animation: slideIn var(--duration-extra-long) ease-out forwards;
```

Comment:
For my personal knowledge, I always assumed anything coming into view is better with `ease-in` in general. Thoughts?

Reply:
Ease-out will slow it down a bit at the end, which feels more "real-life" to me personally. When we're driving our cars for instance, we generally aim to slow before a stop.

Reply:
Thank you!